config/filter.d/qpsmtpd.conf

```diff
@@ -0,0 +1,11 @@
+
+[Definition]
+
+failregex = ^.*535 LOGIN authentication failed for .* from <HOST>$
+ignoreregex =
+
+datepattern = ^%%a %%b %%d %%H:%%M:%%S %%Y
```

Comment:
This `datepattern` is one of default (anchored at begin) patterns of fail2ban, but you can indeed leave it if it is surely always this timestamp format (and not something configurable by system/in syslog).